hedera-mirror-rest-java/src/main/java/com/hedera/mirror/restjava/common/RangeOperator.java

```diff
@@ -36,6 +36,10 @@ public enum RangeOperator {
     private final String operator;
     private final BiFunction<Field, Object, Condition> function;
 
+    public static boolean isInclusive(RangeOperator operator) {
+        return operator == RangeOperator.LT || operator == RangeOperator.GT;
```

Comment:
It doesn't need to be static and the logic is inverted for being inclusive.

```suggestion
    public boolean isInclusive() {
        return this != RangeOperator.LT && operator != RangeOperator.GT;
```

Reply:
Thanks I have fixed this.